AlphaWallet/UI/AddressTextField.swift

```diff
@@ -293,12 +343,27 @@ class AddressTextField: UIControl {
                 }
 
                 guard oldTextValue == strongSelf.textField.text?.trimmed else { return }
-                strongSelf.ensAddressLabel.isHidden = false
-                strongSelf.ensAddressLabel.text = address.address
+
+                strongSelf.resolvedState = .address(value: address.address)
             }
         }
     }
 
+    private func queueEnsResolution(ofValue addressString: String) {
```

Comment:
Can't build without making this non-private.

But is it better if we just keep it private and call it internally, in `AddressTextField.value`'s setter? Otherwise we have to call `queueEnsResolution(ofValue:)` in other places where `AddressTextField` is used, not just `SendViewController`

Reply:
> There's a problem when I perform this sequence in the Send ETH screen:
> 
> 1. Type ENS name
> 2. Address resolves and gets displayed
> 3. Tap clear
> 4. Tap paste (a valid address, the same address as the one that appeared in step 2, if it matters)
> 5. The address doesn't appear in the textbox (it's still empty), but the ENS name appears below in green

@hboon fixed